actions/tomee-dependency/main.go

```diff
@@ -59,6 +61,7 @@ func main() {
 				}
 
 				versions[verKey] = fmt.Sprintf("%s/tomee-%[2]s/apache-tomee-%[2]s-%s.tar.gz", uri, verVal, dist)
+				sources[verKey] = fmt.Sprintf("%s/tomee-%[2]s/tomee-project-%[2]s-source-release.zip", uri, verVal, dist)
```

Comment:
so you got that from an HTML document?

Reply:
That's correct! It's how the binaries are retrieved too